people/durin42.toml

```diff
@@ -0,0 +1,4 @@
+name = 'Augie Fackler'
+github = 'durin42'
+github-id = 20269
+email = 'durin42@gmail.com'
```

Comment:
I prefer raf@durin42.com - if that matters/is feasible (if you need this to be a Google Account then durin42@gmail is correct)

Reply:
Sure!